pandas/core/indexes/timedeltas.py

```diff
@@ -420,7 +421,8 @@ def insert(self, loc, item):
             new_i8s = np.concatenate(
                 (self[:loc].asi8, [item.view(np.int64)], self[loc:].asi8)
             )
-            return self._shallow_copy(new_i8s, freq=freq)
+            tda = type(self._data)._simple_new(new_i8s, freq=freq)
```

Comment:
doen't this copy twice?

Reply:
no, simple_new doesnt make a copy

Reply:
k, that's what i thought, can you (followup ok), document this loudly in the _simple_new